### PR TITLE
Dial the logging down a notch

### PR DESCRIPTION
**Pull request by sachaaaaa**
Disable verbose logging

### CODE REVIEW

File: ts/state/createStore.ts [hunk 1]
```diff
@@ -25,7 +25,8 @@ const logger = createLogger({
 });
 
 // Exclude logger if we're in production mode
-const middlewareList = env === 'production' ? [promise] : [promise, logger];
+const enableLogging = false && env === 'production'; // ALWAYS TURNED OFF
+const middlewareList = enableLogging ? [promise] : [promise, logger];
```
Comment (BeaudanBrown) [inline ts/state/createStore.ts]: Won't this always *enable* the logging?
(I.e. if `enableLogging` is false then `middlewareList = [promise, logger];`?

Comment (Mikunj) [inline ts/state/createStore.ts]:
```suggestion
const middlewareList = !enableLogging ? [promise] : [promise, logger];
```

Comment (sachaaaaa) [reply ts/state/createStore.ts]: 😅 oops

File: ts/state/createStore.ts [hunk 2]
```diff
@@ -25,7 +25,8 @@ const logger = createLogger({
 });
 
 // Exclude logger if we're in production mode
-const middlewareList = env === 'production' ? [promise] : [promise, logger];
+const enableLogging = false && env === 'production'; // ALWAYS TURNED OFF
```
Comment (Mikunj) [inline ts/state/createStore.ts]:
```suggestion
const enableLogging = false && env !== 'production'; // ALWAYS TURNED OFF
```

File: ts/state/createStore.ts (hunk 1 above)
Comment (Mikunj) [reply ts/state/createStore.ts]:
```suggestion
const middlewareList = !enableLogging ? [promise] : [promise, logger];
```

File: ts/state/createStore.ts (hunk 2 above)
Comment (sachaaaaa) [reply ts/state/createStore.ts]: I changed it do disableLogging :)

**Pull request merged**